pkg/sdk/go/sdk.go

```diff
@@ -1001,21 +1018,24 @@ func NewSDK(conf Config) SDK {
 
 // processRequest creates and send a new HTTP request, and checks for errors in the HTTP response.
 // It then returns the response headers, the response body, and the associated error(s) (if any).
-func (sdk mfSDK) processRequest(method, url, token, contentType string, data []byte, expectedRespCodes ...int) (http.Header, []byte, errors.SDKError) {
+func (sdk mfSDK) processRequest(method, url, token string, data []byte, requestHeaders map[string]string, expectedRespCodes ...int) (http.Header, []byte, errors.SDKError) {
 	req, err := http.NewRequest(method, url, bytes.NewReader(data))
 	if err != nil {
 		return make(http.Header), []byte{}, errors.NewSDKError(err)
 	}
 
+	req.Header.Add("Content-Type", string(CTJSON))
```

Comment:
In this case, it is overwritten in the next lines if `Content-Type ` is set in the header parameter. This is because `request.Header.Add()` appends any existing value associated with the key.